GGProject/workflow/urls.py

```diff
@@ -2,8 +2,9 @@
 from . import views
 
 urlpatterns = [
-    path('employees/', views.ListEmployeesView.as_view(), name="employees-all"),
-    path('pictures/', views.ListPicturesView.as_view(), name="pictures-all"),
-    path('temp-photos/', views.ListTempPhotosView.as_view(), name="temp-photos-all"),
-
+    path('employees/', views.ListEmployees),
+    path('employees/<int:emp_ID>/', views.SingleEmployee),
+    path('pictures/', views.ListPictures),
+    path('pictures/<int:emp_ID>/', views.EmployeePictures),
+    path('pictures/<str:name>/', views.SinglePicture),
```

Comment:
The last line of an array should not end in a comma

Reply:
That's how all of the default Django code does it, and also how all of its example documentation does it.

Reply:
Ok that's fine